src/lando/main/migrations/0032_alter_worker_type_multitrainupliftrequest_and_more.py

```diff
@@ -0,0 +1,171 @@
+# Generated by Django 5.2.7 on 2025-10-25 03:28
+
+import django.db.models.deletion
+from django.conf import settings
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("main", "0031_alter_landingjob_requester_email"),
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="worker",
+            name="type",
+            field=models.CharField(
+                choices=[
+                    ("LANDING", "Landing worker"),
+                    ("AUTOMATION", "Automation worker"),
+                    ("UPLIFT", "Uplift worker"),
```

Comment:
We should add this worker type to the dev environment setup, too https://github.com/mozilla-conduit/lando/blob/c156d48d60870f7018436e19ad58aad6fb4f1db9/src/lando/utils/management/commands/setup_dev.py#L18-L26